src/core/ext/filters/client_channel/lb_policy/subchannel_list.h

```diff
@@ -281,7 +282,7 @@ SubchannelData<SubchannelListType, SubchannelDataType>::~SubchannelData() {
 
 template <typename SubchannelListType, typename SubchannelDataType>
 void SubchannelData<SubchannelListType, SubchannelDataType>::
-    UnrefSubchannelLocked(const char* reason) {
+    UnrefSubchannelLocked(const char* /*reason*/) {
```

Comment:
This parameter shouldn't be unused.  It should be logged as part of the message on line 288 below.